packages/framework/src/Component/Router/FriendlyUrl/FriendlyUrlDataFactoryInterface.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Shopsys\FrameworkBundle\Component\Router\FriendlyUrl;
 
+/**
+ * @method FriendlyUrlData createFromIdAndName(int $id, string $name)
```

Comment:
It's to maintain backward compatibility? Can you please mark it somehow, so when we remove deprecations we don't forget to update the interface as this should definitely be part of the interface
(and I don't really know how properly)

Reply:
yes, for BC. I believe the @method annotation is sufficient enough - I have added a note to the user story that solves deprecations for 8.0